src/cpu/core_normal/prefix_none.h

```diff
@@ -495,6 +495,7 @@
 			//Little hack to always use segprefixed version
 			BaseDS=BaseSS=0;
 			GetRMrw;
+			if (rm >= 0xc0) goto illegal_opcode;   // Direct register causes #UD exception
```

Comment:
Connecting the dots between the code ">= 0xc0" and comment "Direct register".

Maybe something like:
```c++
constexpr uint8_t start_of_direct_registers = 0xc0;
if (rm >= start_of_direct_registers)
        goto illegal_opcode;
```
Would be more self-documenting (ie: `rm` is now allowed to be any of the direct registers?)